cl/corpus_importer/tasks.py

```diff
@@ -867,18 +888,12 @@ def upload_to_ia(
 
 
 @app.task
-def mark_court_done_on_date(
-    status: int, court_id: str, d: date
-) -> Optional[int]:
-    court_id = map_pacer_to_cl_id(court_id)
+def mark_court_done_on_date(log_id: int, status: int) -> Optional[int]:
     try:
-        doc_log = PACERFreeDocumentLog.objects.filter(
-            status=PACERFreeDocumentLog.SCRAPE_IN_PROGRESS, court_id=court_id
-        ).latest("date_queried")
+        doc_log = PACERFreeDocumentLog.objects.get(pk=log_id)
     except PACERFreeDocumentLog.DoesNotExist:
```

Comment:
how are we marking courts done without identifying the court used

Reply:
the PACERFreeDocumentLog model contains the court that was scraped or is being scraped, this object is created in an early process and we only care about updating it if it is in progress, successful or failed.

We create it in mark_court_in_progress function